packages/atomic/src/utils/accessibility-utils.tsx

```diff
@@ -75,8 +75,10 @@ export function FocusTarget() {
           element = el;
           if (focusOnNextTarget) {
             focusOnNextTarget = false;
-            element.focus();
-            onFocusCallback?.();
+            setTimeout(() => {
+              el.focus();
+              onFocusCallback?.();
+            });
```

Comment:
Same as https://github.com/coveo/ui-kit/pull/2078#discussion_r894688354

I noticed some inconsistencies with the focus on this component too, after all.